ui-v2/app/components/tabular-collection.js

```diff
@@ -37,6 +56,21 @@ const change = function(e) {
     const value = e.currentTarget.value;
     if (value != get(this, 'checked')) {
       set(this, 'checked', value);
+      if (e.currentTarget.getAttribute('id') !== 'actions_close') {
```

Comment:
So this keeps component state on the DOM via an id of `actions_close`?

Reply:
You probably need a little more context to follow this, feel free to ping me and I can walk you through it.

TLDR; It doesn't run the code/fork if the selected radio button on change is the 'reset radiogroup' (`id="actions_close"`) button.

Basically there is an `<input type="radio" id="actions_close" />` that is hidden in the `<th>` of the 'actions column' in the table. All of the other dropdowns use the same radiobutton `name=""`/radiogroup. This ensures you can only ever have one dropdown visible at once, and I have a way to 'unset' the group (the 'actions_close' radio essentially 'resets' the radiogroup). 

The above code needs to run whenever a dropdown menu is opened, in other words, whenever the radiogroup selection changes. The thing is, the 'action_close' radio is in the group also, so if the action_close button was selected/clicked (in other words I didn't open a dropdown, but reset the group/ensured they are all closed), don't run the code.

Sorry probably not the best explanation, ping me if you want more info.